geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegion.java

```diff
@@ -1842,11 +1846,6 @@ public boolean getOffHeap() {
     return this.offHeap;
   }
 
-  @Override
-  public boolean isConcurrencyChecksEnabled() {
```

Comment:
For booleans, I actually prefer methods with names like "isThing" rather than "getThing" in most cases, but that is purely a personal preference.

Reply:
We found that we had two existing methods: isConcurrencyChecksEnabled and getConcurrencyChecksEnabled. We wanted to change to only have one accessor method. The "get" flavor not because it was better but because we have the external interface RegionAttributes that AbstractRegion implements and has the "get" flavor.

Reply:
Sounds reasonable to me. Thanks for the reasoning behind the decision.